google/cloud/aiplatform/prediction/local_model.py

```diff
@@ -261,7 +259,7 @@ def create_cpr_model(
             health_route=DEFAULT_HEALTH_ROUTE,
         )
 
-        return cls(container_spec)
+        return cls(serving_container_spec=container_spec)
 
     def upload(
```

Comment:
This doesn't use the mixin, the problem with two competing `upload` methods still exists.

If you're going the route of giving the mixin's `upload` method no implementation at all, then make the mixin an abstract class to enforce that it's method signature gets used. `LocalModel.upload` can call `LocalModel._upload` to make use of instance properties if necessary.

Reply:
Removed upload method from LocalModel.